env.js

```diff
@@ -46,6 +46,78 @@ function config ( ) {
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
 
   env.enable = readENV('ENABLE');
+
+  env.defaults = { // currently supported keys must defined be here
+	  'units': 'mg/dL'
+	, 'timeFormat': '12'
+	, 'nightMode': false
+	, 'showRawbg': 'never'
+	, 'customTitle': 'Nightscout'
+	, 'theme': 'default'
+    , 'alarmUrgentHigh': true
```

Comment:
Check whitespace, looks like mix of tabs and space.  It bugs me that all the files use different whitespace conventions, I'd reformat it all, but don't want to deal with the merge conflicts